### ISSUE TITLE
Scroll Bar Disappears on Chrome

### ISSUE THREAD

**Issue by oldgristmill** [mcguirepr89/BirdNET-Pi#933]
New install on a RP4 and there are no vertical scrollbars on any of the pages served by Birdnet-Pi. Same issue seems to exist with Microsoft Edge. Haven't checked on Safari or other browsers that use a different rendering engine.

Can anyone verify this problem? Is it the browser or the structure of the pages being served?

**Comment by lloydbayley** [comment]
When you say scroll bar disappears, was it ever there? Did it suddenly disappear?
Also, does the page scroll if you use the mouse wheel or drag etc etc?
I'm not able to replicate this on any browser/operating system. Very odd!

**Comment by jaredb7** [comment]
I don't think I've noticed a scrollbar, but when I check... it's only there for Firefox but not Chrome or Edge. 
Not sure if this is by design?, the no scrollbars I mean.

Chrome and Edge use the same rendering engine so it could be a browser thing?

![image](https://github.com/mcguirepr89/BirdNET-Pi/assets/799998/8d508c21-6d01-4eb4-9a3e-70a493ee8b8c)
![image](https://github.com/mcguirepr89/BirdNET-Pi/assets/799998/0137a82d-a075-4668-ab69-186b21262013)
![image](https://github.com/mcguirepr89/BirdNET-Pi/assets/799998/0faa7eda-21de-43c3-a827-1d4585c40b05)

**Comment by oldgristmill** [comment]
@jaredb7, thanks for confirming that I'm not crazy! I have no idea how to figure out whether it's a bug with the Chrome render engine or with the html/css/js being output by birdnet-pi.

**Comment by ehpersonal38** [comment]
Yes all scrollbars are disabled, but on Firefox and IE it wasn't working. I've just done a commit which removes them. Thanks!

**Comment by ehpersonal38** [comment]
Patrick removed them long ago, probably for aesthetic reasons. But I now agree with you that they can be quite useful so I've added them back in a new commit - 5336736b38143d7f73dccd707de126fd53b8efad.
